### ISSUE TITLE
Cluster: Fix error with wazuh-clusterd daemon

### ISSUE THREAD

**Issue by davidjiglesias** [wazuh/wazuh#3264]
There is an issue in `3.10` branch with the cluster due to some changes introduced in 56efe53f6f6daf160cc565cc5d74d3cc506de805.

When trying to run wazuh-manager service:
```
root@359a2828ea43:/# service wazuh-manager restart
wazuh-clusterd not running...
wazuh-modulesd not running...
ossec-monitord not running...
ossec-logcollector not running...
ossec-remoted not running...
ossec-syscheckd not running...
ossec-analysisd not running...
ossec-maild not running...
ossec-execd not running...
wazuh-db not running...
ossec-authd not running...
ossec-agentlessd not running...
ossec-integratord not running...
ossec-dbd not running...
ossec-csyslogd not running...
Wazuh v3.10.0 Stopped
Traceback (most recent call last):
  File "/var/ossec/framework/scripts/wazuh-clusterd.py", line 12, in <module>
    from wazuh.cluster import cluster, __version__, __author__, __ossec_name__, __licence__, master, local_server, worker
  File "/var/ossec/framework/python/lib/python3.7/site-packages/wazuh-3.10.0-py3.7.egg/wazuh/cluster/master.py", line 20, in <module>
    from wazuh.cluster.dapi import dapi
  File "/var/ossec/framework/python/lib/python3.7/site-packages/wazuh-3.10.0-py3.7.egg/wazuh/cluster/dapi/dapi.py", line 11, in <module>
    from wazuh.cluster import local_client, cluster, common as c_common
  File "/var/ossec/framework/python/lib/python3.7/site-packages/wazuh-3.10.0-py3.7.egg/wazuh/cluster/local_client.py", line 8, in <module>
    from wazuh.cluster import client, cluster
  File "/var/ossec/framework/python/lib/python3.7/site-packages/wazuh-3.10.0-py3.7.egg/wazuh/cluster/client.py", line 13, in <module>
    class AbstractClientManager:
  File "/var/ossec/framework/python/lib/python3.7/site-packages/wazuh-3.10.0-py3.7.egg/wazuh/cluster/client.py", line 47, in AbstractClientManager
    def add_tasks(self) -> List[asyncio.coroutine, Tuple]:
  File "/var/ossec/framework/python/lib/python3.7/typing.py", line 251, in inner
    return func(*args, **kwds)
  File "/var/ossec/framework/python/lib/python3.7/typing.py", line 627, in __getitem__
    _check_generic(self, params)
  File "/var/ossec/framework/python/lib/python3.7/typing.py", line 205, in _check_generic
    raise TypeError(f"Too {'many' if alen > elen else 'few'} parameters for {cls};"
TypeError: Too many parameters for typing.List; actual 2, expected 1
wazuh-clusterd: Configuration error. Exiting
```

**Comment by davidjiglesias** [comment]
Status update
## Summary
Today I have fixed the issue with the cluster in branch 3.10. A change introduced in 56efe53f6f6daf160cc565cc5d74d3cc506de805 added
 `-> List[asyncio.coroutine,Tuple]`, but the correct return typing is `List[Tuple[asyncio.coroutine,Tuple]]`

```
    def add_tasks(self) -> List[Tuple[asyncio.coroutine,Tuple]]:
        if self.performance_test:
            task = self.client.performance_test_client, (self.performance_test,)
        elif self.concurrency_test:
            task = self.client.concurrency_test_client, (self.concurrency_test,)
        elif self.file:
            task = self.client.send_file_task, (self.file,)
        elif self.string:
            task = self.client.send_string_task, (self.string,)
        else:
            return []

        return [task]
```

## Tasks
* [3h] Debug and find what was causing the issue. Performed several test with 3.9 and 3.10 and checked code differences.
* [1h] Fix and test wazuh

## Difficulties
* I checked and composed different docker environments to test 3.9 and 3.10 branches in order to debug and find the issue.
## Tests performed
* Run wazuh-manager after fix:
```
root@655a4a453514:/# service wazuh-manager restart
Killing wazuh-clusterd...
Killing wazuh-modulesd...
Killing ossec-monitord...
Killing ossec-logcollector...
Killing ossec-remoted...
Killing ossec-syscheckd...
Killing ossec-analysisd...
ossec-maild not running...
Killing ossec-execd...
Killing wazuh-db...
Killing ossec-authd...
ossec-agentlessd not running...
ossec-integratord not running...
ossec-dbd not running...
ossec-csyslogd not running...
Wazuh v3.10.0 Stopped
Starting Wazuh v3.10.0...
Started ossec-csyslogd...
Started ossec-dbd...
2019/05/07 15:33:01 ossec-integratord: INFO: Remote integrations not configured. Clean exit.
Started ossec-integratord...
Started ossec-agentlessd...
Started ossec-authd...
Started wazuh-db...
Started ossec-execd...
Started ossec-analysisd...
Started ossec-syscheckd...
Started ossec-remoted...
Started ossec-logcollector...
Started ossec-monitord...
Started wazuh-modulesd...
Started wazuh-clusterd...
Completed.
```
## Pending

**Comment by davidjiglesias** [comment]
Issue fixed in #3266
